projects/core/src/i18n/i18next/i18next-plugins/i18next-logger-plugin.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from '@spartacus/core';
+import { LoggerModule } from 'i18next';
+import { I18NEXT_LOGGER_PLUGIN } from './i18next-logger-plugin';
+
+describe('i18next logger plugin', () => {
+  let plugin: LoggerModule;
+  let logger: LoggerService;
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [LoggerService],
+    });
+
+    plugin = TestBed.inject(I18NEXT_LOGGER_PLUGIN);
+    logger = TestBed.inject(LoggerService);
+  });
+
+  it('should return logger plugin', () => {
+    expect(plugin).toBeTruthy();
+    expect(plugin.type).toEqual('logger');
+  });
+
+  it('should log', () => {
+    const log = spyOn(logger, 'log');
+
+    plugin.log(['test']);
+
+    expect(log).toHaveBeenCalledWith('test');
+  });
```

Comment:
[MINOR]

Let's add some test with multiple arguments

Reply:
done in https://github.com/SAP/spartacus/pull/17538